pkgs/by-name/st/starpls/package.nix

```diff
@@ -1,57 +1,49 @@
 {
+  rustPlatform,
   lib,
-  stdenv,
-  fetchurl,
-  autoPatchelfHook,
   testers,
   starpls,
-}:
-
-let
-  manifest = lib.importJSON ./manifest.json;
+  fetchFromGitHub,
+  protobuf,
+}: let
+  commit = "db21acd3cb24893315dd601484c7d40689589e9a";
```

Comment:
This is the most recent commit but I couldn't tell if there was a different version on the main repository.
I've kept it to the same but added the commit specifier.